README.md

```diff
@@ -1,4 +1,681 @@
+	getMirroringTopicSelectionOptionsModel.Headers = map[string]string{"x-custom-header": "x-custom-value"}
+
+	// Enable retries.
+	serviceAPI.EnableRetries(0, 0)
+
+	// Call GetMirroringTopicSelection.
+	result, response, operationErr := serviceAPI.GetMirroringTopicSelection(getMirroringTopicSelectionOptionsModel)
+	if operationErr != nil {
+		return fmt.Errorf("Error Listing Mirroring Topics: %s\n", operationErr.Error())
+	}
+
+	// Check the result.
+	if response.StatusCode != http.StatusAccepted {
+		return fmt.Errorf("Error Listing Mirroring Topics: status %d\n", response.StatusCode)
+	}
+
+	// Loop and print mirroring topics.
+	for _, topicName := range result.Includes {
+		fmt.Printf("\tname: %s\n", topicName)
+	}
+```
+
+### Replace selection of topics which are mirrored
+
+Replace mirroring topic selection
+
+Mirroring user controls are available on the target cluster in a mirroring environment.
+
+To replace the current topic selection, issue a POST request to /admin/mirroring/topic-selection
+
+Expected status codes
+
+    200: Replaced topic selection successfully. The new selection is returned in following format:
+
+	{
+	"includes": [
+		"^prefix1_.*",
+		"^prefix2_.*"
+	]
+	}
+
+    400: Invalid request. The request data cannot be parsed and used to replace the topic selection.
+    403: Unauthorized to use mirroring user controls.
+    404: Mirroring not enabled. The mirroring user control apis are only available on a target in a pair of clusters with mirroring enabled between them.
+    415: Unsupported media type. Content-Type header with application/json is required.
+    503: An error occurred handling the request.
+
+#### Example
+
+```golang
+	replaceMirroringTopicSelectionOptionsModel := new(ibmeventstreamsadminrestapiv1.ReplaceMirroringTopicSelectionOptions)
+	replaceMirroringTopicSelectionOptionsModel.Includes = []string{"test-topic"}
+	replaceMirroringTopicSelectionOptionsModel.Headers = map[string]string{"x-custom-header": "x-custom-value"}
+
+	// Enable retries.
+	serviceAPI.EnableRetries(0, 0)
+
+	// Invoke operation with valid options model (positive test)
+	result, response, operationErr := serviceAPI.ReplaceMirroringTopicSelection(replaceMirroringTopicSelectionOptionsModel)
+	if operationErr != nil {
+		return fmt.Errorf("Error Replacing Mirroring Topics: %s\n", operationErr.Error())
+	}
+
+	// Check the result.
+	if response.StatusCode != http.StatusAccepted {
+		return fmt.Errorf("Error Replacing Mirroring Topics: status %d\n", response.StatusCode)
+	}
+
+	// Loop and print mirroring topics.
+	for _, topicName := range result.Includes {
+		fmt.Printf("\ttopic added: %s\n", topicName)
+	}
+	
+```
+
+### List active mirroring topics
+
+Mirroring user controls are available on the target cluster in a mirroring environment.
+
+To get the list of currently mirrored topics, issue an GET request to /admin/mirroring/active-topics
+
+Expected status codes
```

Comment:
This does not render correctly, see https://github.com/IBM/eventstreams-go-sdk/blob/980eb5aa082f35ca4929fbe655d6806cb93d54eb/README.md#list-active-mirroring-topics

Reply:
fixed